spec/integration/session_spec.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 # coding: utf-8
```

Comment:
The `coding` pragma must be on the first line, otherwise it is ignored

Reply:
Good to know that, thanks!